cpu/numatop.py

```diff
@@ -79,11 +80,12 @@ def test(self):
         # Analyse record file for mgen record
         with open('%s/result_file' % self.sourcedir, 'r') as f_read:
             lines = f_read.readlines()
-            if 'PID' in lines[2]:
-                if 'mgen' not in lines[3]:
-                    self.fail(
-                        'Numatop failed to record mgen latency. Please check '
-                        'the record file: %s/result_file' % self.sourcedir)
+            for line in lines:
+                if 'mgen' in line.split():
```

Comment:
is `split()` needed here ?

Reply:
not necessarily, but thought it looks better to have check for the application. than the entire string.

Reply:
@narasimhan-v right. I think it does not make sense to split at all. I ll make change as you have suggested.